src/main/java/de/zalando/aruha/nakadi/FlowIdRequestFilter.java

```diff
@@ -0,0 +1,47 @@
+package de.zalando.aruha.nakadi;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+
+public class FlowIdRequestFilter implements Filter {
+    public static final String X_FLOW_ID_HEADER = "x-flow-id";
```

Comment:
`httpServletRequest.getHeader()` is case-insensitive.
But yes, to avoid confusion I should change it.